### DISCUSSION TITLE
Mac

### DISCUSSION THREAD

**Discussion by oasissan** [enjoysmath/BootstrapStudioToDjango#4] [General]
I really needed this but I am using mac. Great job btw. I hope you do it for mac someday thank you.

**Comment by enjoysmath** [comment]
... It's python.  It is for mac already

**Reply by oasissan** [reply]
Not sure how to run it. Can provide instructions on how to run it? Thank you

**Comment by enjoysmath** [comment]
See instructions on Code page of this repository :)

EnjoysMath

**Reply by oasissan** [reply]
Export Script "/Users/kill_390/Desktop/BootstrapStudioToDjango-master/dev_bss_to_django.exe" resulted in one or more errors
Script /Users/kill_390/Desktop/BootstrapStudioToDjango-master/dev_bss_to_django.exe couldn't be run

I saw the instructions before. How can exe work in mac?

**Comment by enjoysmath** [comment]
Instead you want to compile everything for Mac.  Simply create a
dev_bss_to_django.sh or shell script you can run on Mac.   I don't have
time this week to go into auto-py-to-exe and generate a Mac ver and test
that it works..

You need something to call python <enry point> on my script because BSS
doesn't take command-line args I don't think.

EnjoysMath

**Reply by geraudch** [reply]
Hi @kill390, 

Have you made it work? Is there anything you can share? 
I'd love to make this export script as part of my toolchain!

**Reply by enjoysmath** [reply]
@geraudch 

See my repository.  Mine even has a GUI

**Reply by oasissan** [reply]
https://htmltodjango.com/ 
I have made this site its much simpler but it does most of the job

**Reply by enjoysmath** [reply]
That's awesome!   But what about a continuous workflow from BSS to Django?  Does it support that?

I.e. make a small edit in BSS, hit export, then refresh page in browser.

**Reply by oasissan** [reply]
clone this repo 
https://github.com/kill390/html2django/tree/main

then run this command `chmod +x html2django/process_file.py` 
then add process_file.py as export script this will work for mac and linux
